core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMGraph.java

```diff
@@ -29,6 +30,7 @@ public class STDCMGraph implements Graph<STDCMNode, STDCMEdge> {
     public final RollingStock rollingStock;
     public final RollingStock.Comfort comfort;
     public final double timeStep;
+    HashMap<BlockSimulationParameters, Envelope> simulatedEnvelopes;
```

Comment:
I would have preferred to have the map inside `STDCMSimulations` (which wouldn't be static anymore), with an instance of `STDCMSimulations` here. Functionally it's the same, but it keeps anything related to simulation in one file.

Reply:
Can and will do.